### DISCUSSION TITLE
How do I set cur dir on exit?

### DISCUSSION THREAD

**Discussion by glyh** [sayanarijit/xplr#576] [Q&A] [answered]
As title, thanks!

**Comment by sayanarijit** [comment, marked as answer]
https://xplr.dev/en/awesome-hacks#cd-on-quit

**Reply by glyh** [reply]
I think this does not work with default current dir:
```fish
xplr --print-pwd-as-result /tmp
```
the above command doesn't print anything as output

**Reply by sayanarijit** [reply]
Do you quit by pressing `enter` or `q`?

**Reply by glyh** [reply]
Yeah I did, here's the [issue](https://github.com/sayanarijit/xplr/issues/578#issue-1561072658) I opened
